src/components/profile/edit/EditSlack.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <div :class="{ 'slack-container': true, 'sign-in': slack.v === '' }">
+    <a class="sign-in-link" v-if="slack.v === ''" href="/whoami/slack/add">
+      <img
+        alt="Sign in with Slack"
+        src="https://platform.slack-edge.com/img/sign_in_with_slack.png"
```

Comment:
those will be blocked by our CSP headers. Can we self host them?

Reply:
Completed.